docs/community/faqs.md

```diff
@@ -1,34 +1,42 @@
 ---
 id: faqs
-title: 'Frequently asked Questions'
-sidebar_label: 'FAQS'
+title: 'Frequently Asked Questions'
+sidebar_label: FAQs
 keywords:
   - faqs
 ---
 
+## Contributions
+
 1. **How do I find good first issues?**
-We believe that [good first issues don't exist](https://opensauced.pizza/blog/good-first-issues-dont-exist), and the best way to find an issue to work on is to create the issue yourself. 
-
-  However, sites like [good first issues](https://goodfirstissues.com/) and [first timers only](https://www.firsttimersonly.com/) can lead you to projects and issues worth working on.  You can also use the `good first issue` label in the search engine of the project you want to contribute to. Check out [this guide](https://www.freecodecamp.org/news/how-to-find-good-first-issues-on-github/) to learn more. 
-  
-  **Additional resources**:
-  
-  - ["Who's looking for open source contributors?"](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
-  - [Intro to Open Source Course](https://intro.opensauced.pizza/)
-  
+
+   We believe that [good first issues don't exist](https://opensauced.pizza/blog/good-first-issues-dont-exist), and the best way to find an issue to work on is to create the issue yourself.
+
+   However, sites like [Good First Issues](https://goodfirstissues.com/) and [First Timers Only](https://www.firsttimersonly.com/) can lead you to projects and issues worth working on. You can also use the `good first issue` label in the search engine of the project you want to contribute to. Check out [this guide](https://www.freecodecamp.org/news/how-to-find-good-first-issues-on-github/) to learn more.
+
+   **Additional resources**:
+
+   - ["Who's looking for open source contributors?"](https://dev.to/opensauced/whos-looking-for-open-source-contributors-week-46-247i)
+   - [Intro to Open Source Course](https://intro.opensauced.pizza/)
+
 2. **My contribution does not show up on my OpenSauced profile. How do I fix it?**
-Create an [insight page](https://app.opensauced.pizza/hub/insights) or add your merged pull request to [a highlight](https://app.opensauced.pizza/feed).
+
+   Create an [insight page](https://app.opensauced.pizza/hub/insights) or add your merged pull request to [a highlight](https://app.opensauced.pizza/feed).
```

Comment:
- I feel like the answer is not answering the question here because creating an insight page is different with creating a highlight.
- We do have a "Contributions" tab on our profile. Does the intention of the question is to ask about how to fix if contributions are not showing up in the "Contribution" tab or about how to fix if the contributions are not showing on the "Highlights" tab?

We might want to consider to update this with something like below or remove it.

```markdown
2. **My contribution does not appear on the "Highlights" tab in my OpenSauced profile.**

   You need to first create [a highlight](https://app.opensauced.pizza/feed) of your pull request, issue, or blog post on Dev.to.
```

Reply:
![image](https://github.com/open-sauced/docs/assets/34313413/ccdbfa51-ec5d-4eef-a81d-8e40a9e06257)

It's an issue of when we're getting data to populate that chart. If you create a highlight with the repo or an insight page with the repo, it will make the update. We've got something in the works that should fix this, but right now this is the solution.